resources/measures/HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -2904,21 +2929,18 @@ def self.apply_cooling_setpoints(model, runner, weather, clg_wkdy_monthly, clg_w
     finished_zones.each do |finished_zone|
       thermostat_setpoint = finished_zone.thermostatSetpointDualSetpoint
       if thermostat_setpoint.is_initialized
-
         thermostat_setpoint = thermostat_setpoint.get
         thermostat_setpoint.setHeatingSetpointTemperatureSchedule(heating_setpoint.schedule)
         thermostat_setpoint.setCoolingSetpointTemperatureSchedule(cooling_setpoint.schedule)
-
+        
```

Comment:
I'd try to maintain the same tabs/spacing/newlines as before. This will make diffing much easier.